src/common/test/test_debugger_communication_client.py

```diff
@@ -3,10 +3,25 @@
 from unittest import mock
 import socketio
 import threading
+import os
+import sys
 
-from adafruit_circuitplayground import express
-from common import debugger_communication_client
 from common import constants as CONSTANTS
+
+abs_path_to_parent_dir = os.path.join(
+    os.path.dirname(os.path.abspath(__file__)), "..", ".."
+)
+
+sys.path.insert(0, abs_path_to_parent_dir)
+# Insert absolute path to Micropython libraries for micro:bit into sys.path
```

Comment:
I would put this comment above the line it's referring to (or maybe remove the line break on line 20).
Now it seems as if it's referring to the next three lines where we calculate the path.